pkg/common/core_validations_test.go

```diff
@@ -0,0 +1,38 @@
+package common
+
+import (
+	"testing"
+
+	"github.com/mitchwebster/botblitz/pkg/common/pb"
+)
+
+func TestLandscapeValidations(t *testing.T) {
+
+	team := &pb.FantasyTeam{
+		Id:    "001",
+		Name:  "Test Team",
```

Comment:
nit: all test data should be zany and esoteric otherwise how will people know we're super chill and laid back software engineers.

Reply:
Lol tried to make it a little more zany, but certainly open to suggestions